### PR TITLE
remove CLI.SetupNamespace() in favour of CLI.SetupProject()

### PR DESCRIPTION
**Pull request by stlaz**
/assign @openshift/sig-networking 
/assign @openshift/openshift-team-network-edge-maintainers

### CODE REVIEW

**Comment by abhat** [comment]
LGTM. @danwinship can you lgtm?

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Comment by soltysh** [comment]
/hold
for @danwinship

File: test/extended/networking/egressip.go [hunk 1]
```diff
@@ -154,11 +154,6 @@ var _ = g.Describe("[sig-network][Feature:EgressIP]", func() {
 			targetPort)
 
 		g.By("Creating a project for the prober pod")
-		// Create a target project and assign source and target namespace
```
Comment (andreaskaris) [inline test/extended/networking/egressip.go:157]: nit: Why remove that comment, is it wrong or imprecise?

Comment (andreaskaris) [inline test/extended/networking/egressip.go:157]: nit: can you keep:
~~~
	// Create a target project and assign source and target namespace
Attach files by dragging & dropping, selecting or pasting them.
		// to variables for later use.
~~~

**Comment by stlaz** [comment]
/unhold
as Andreas reviewed for egress

**Pull request merged**